geopandas/io/tests/test_io.py

```diff
@@ -54,6 +54,13 @@ def test_read_file(self):
         lower_columns = [c.lower() for c in self.columns]
         assert (df.columns[:-1] == lower_columns).all()
 
+    @pytest.mark.webtest
```

Comment:
maybe just "web" instead of "webtest" ? 
But it's fine to add a mark like this, we don't have anything set up like this yet.

Reply:
Good suggestion, had just taken it from the py.test example in their docs.